### PR TITLE
Add a new section for handy adb shell commands.

### PR DESCRIPTION
**Pull request by ZahidRasheed**

### CODE REVIEW

**Comment by nisrulz** [comment]
@ZahidRasheed this looks good to me, except I think it would be nice to have it in a tabular form since all these terminal commands are one liner. Do you think you can convert all of this in a table form?

Thank you

**Comment by ZahidRasheed** [comment]
Sure I can do that :)

File: Readme.md [hunk 1]
```diff
@@ -168,6 +171,119 @@ This one needs special mention due to how useful it is. It basically is a window
...
+	adb shell pm revoke your.app.package android.permission.WRITE_EXTERNAL_STORAGE
+	  
+	```
++ Broadcast Actions
+  	  
+	```sh
+	adb shell am broadcast -a 'my_action' 
+	  
+	```
++ [Simulating Android killing your app in the background] (https://twitter.com/Jahnold/status/759775495655333888)
```
Comment (nisrulz) [inline Readme.md]: There is an extra space between `]` and `(` , which makes the link not render properly. The extra space should be removed.

Comment (ZahidRasheed) [reply Readme.md]: Oh yeah. I'll fix :)

**Pull request merged**